backend/src/apiserver/metadata/metadata_store.go

```diff
@@ -0,0 +1,127 @@
+package metadata
+
+import (
+	"encoding/json"
+	"ml_metadata/metadata_store/mlmetadata"
+	mlpb "ml_metadata/proto/metadata_store_go_proto"
+	"strings"
+
+	argoWorkflow "github.com/argoproj/argo/pkg/apis/workflow/v1alpha1"
+	"github.com/golang/protobuf/jsonpb"
+	"github.com/golang/protobuf/proto"
+	"github.com/kubeflow/pipelines/backend/src/common/util"
+)
+
+// Store encapsulates a ML Metadata store.
+type Store struct {
```

Comment:
should this be called MetadataStore? Store seems generic. WDYT?

Reply:
Actually, it should be Store. The package name is metadata, so clients always refer to it as metadata.Store. See 
http://go/gh/golang/go/wiki/CodeReviewComments#package-names
https://golang.org/doc/effective_go.html#package-names